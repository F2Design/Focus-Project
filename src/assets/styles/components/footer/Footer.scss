/* Primario = #6BBAA7 */
/* Secundario = #FBA100 */
/* Otro = #6C648B */

body {
    font-family: "Source Sans Pro", sans-serif;
    margin: 0;
    background-color: grey;
  }
  
  h1,
  h2,
  h3,
  h4,
  h5,
  h6 {
    font-family: "Open Sans", sans-serif;
    font-weight: normal;
  }
  
  h1 {
    font-size: 32px;
    font-weight: 600;
    line-height: 1.5;
    letter-spacing: -0.2px;
    color: #6bbaa7;
    margin: 0;
  }
  
  h2 {
    color: #026fff;
    text-transform: uppercase;
    font-family: "Source Sans Pro", sans-serif;
    font-weight: bold;
    font-size: 24px;
    margin-bottom: 40px;
  }
  
  .header {
    width: 100%;
    background-image: url("../images/Banner.jpg");
    height: 350px;
  }
  
  #video_background {
    position: absolute;
    width: 100%;
    height: 350px;
    z-index: -1000;
    overflow: hidden;
  }
  #video_pattern {
    background: #fff;
    position: fixed;
    opacity: 0.8;
    left: 0px;
    top: 0px;
    z-index: 1;
  }
  
  .header a {
    color: white;
    text-decoration: none;
  }
  
  .header a {
    display: flex;
    align-items: center;
    padding: 0 10px;
  }
  
  .nav-header {
    width: 100%;
    display: flex;
    align-items: center;
    justify-content: space-between;
  }
  
  .container {
    max-width: 1000px;
    text-align: center;
    margin: 0 auto;
  }
  
  .principal-text {
    margin-top: 50px;
    margin-bottom: 50px;
  }
  
  .filter .input-search-text {
    display: flex;
    justify-content: center;
  }
  
  .filter-rank > input {
    width: 400px;
    border-radius: 0 5px 5px 0;
    height: 25px;
  }
  
  .input-search-text {
    justify-items: center;
    align-items: center;
  }
  
  .input-search-text {
    float: left;
    width: 12%;
    background-color: #fba100;
    font-size: 13px;
    height: 30px;
    border-radius: 5px 0 0 5px;
  }
  
  .input-search {
    display: inline-block;
    margin: auto;
    border-radius: 0 5px 5px 0;
    height: 25px;
  }
  
  .search {
    width: 100%;
    height: 30px;
    position: relative;
    display: flex;
  }
  
  .searchTerm {
    width: 100%;
    border: 1px solid white;
    border-right: none;
    padding: 5px;
    border-radius: inherit;
  }
  
  .searchTerm:focus {
    color: grey;
  }
  
  .searchButton {
    width: 40px;
    height: 30px;
    border: 1px solid white;
    background: white;
    text-align: center;
    color: #fff;
    border-radius: 0 5px 5px 0;
    cursor: pointer;
    font-size: 20px;
  }
  
  /*Resize the wrap to see the search bar change!*/
  .wrap {
    width: 55%;
    float: left;
  }
  
  .ranking-form {
    width: 30%;
    height: 30px;
    border-radius: 5px;
  }
  
  .menu ul {
    display: flex;
    padding: 0;
    list-style: none;
  }
  
  .menu {
    width: 100%;
    display: flex;
    justify-content: space-between;
    align-items: center;
  }
  
  .menu-items {
    width: 500px;
    display: flex;
    height: 40px;
    justify-content: space-evenly;
    align-items: center;
    background-color: rgba(256, 256, 256, 0.2);
    border-radius: 5px;
  }
  
  .figure {
    margin: 0;
    align-items: center;
    justify-content: center;
  }
  .social {
    display: flex;
    margin-left: 50px;
    justify-content: center;
  }
  
  .social__icon {
    line-height: 0;
    margin-right: 20px;
  }
  
  .social__text-footer p {
    margin-left: -10px;
    margin-right: 10px;
  }
  
  .social-menu-icon {
    display: flex;
    align-items: flex-end;
  }
  
  .footer {
    display: grid;
    width: 100%;
    position: absolute;
    bottom: 0;
    grid-template: 40px 40px / repeat(4, minmax(200px, 1fr));
    column-gap: 10px;
    background-color: #6c648b;
    font-size: 16px;
    border-top: 1px solid #ccc;
    color: white;
  }
  
  .item__menu-footer {
    width: auto;
    padding: 5px;
    text-align: center;
  }
  
  .footer ul {
    list-style: none;
  }
  
  .logo__footer {
    margin-left: 20px;
    margin-top: 10px;
  }
  
  .social__footer {
    font-size: 13px;
    margin-top: 20px;
  }
  
  .footer a {
    color: white;
    text-decoration: none;
  }
  
  .social__footer img {
    margin-left: 10px;
  }
  
  .social-link {
    display: inline-block;
    width: 50px;
    height: 50px;
    margin: 0 10px;
    background-size: 50px 50px;
  }
  
  .social-link.facebook {
    background-image: url("../images/facebook.svg");
  }
  
  .social-link.github {
    background-image: url("../images/github.svg");
  }
  