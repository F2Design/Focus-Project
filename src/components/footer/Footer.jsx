import React from 'react';
import '../../assets/styles/components/footer/Footer.scss';
import Logo from '../../assets/images/logo-focus-footer.svg'
import GithubLogo from '../../assets/images/github.svg'
import LinkedinLogo from '../../assets/images/linkedin.svg'
import FacebookLogo from '../../assets/images/facebook.svg'

const Footer = () => (
  <footer className="footer">
    <ul className="logo__footer">
      <figure>
        <a href="index.html">
          <img
            src={Logo}
            height="30"
            alt="Logo Focus"
          />
        </a>
      </figure>
    </ul>

    <ul className="item__menu-footer">
      <li>
        <a href="">Soporte</a>
      </li>
      <li>
        <a href="">Nosotros</a>
      </li>
    </ul>

    <ul className="item__menu-footer">
      <li>
        <a href="">Noticias</a>
      </li>
      <li>
        <a href="">Privacidad</a>
      </li>
    </ul>

    <ul className="social__footer">
      <div className="social">
        <li className="social__icon">
          <a href='#'>
            <img className="" src={GithubLogo} width="20" alt="GitHub"
            />
          </a>
          <p className="social__text-footer">Github</p>
        </li>
        <li className="social__icon">
          <a href="#">
            <img
              className=""
              src={LinkedinLogo}
              width="20"
              alt="Linkedin"
            />
          </a>
          <p className="social__text-footer">Linkedin</p>
        </li>
        <li className="social__icon">
          <a href="#">
            <img
              className=""
              src={FacebookLogo}
              width="20"
              alt="Facebook"
            />
          </a>
          <p className="social__text-footer">Facebook</p>
        </li>
      </div>
    </ul>
  </footer>
);

export default Footer;