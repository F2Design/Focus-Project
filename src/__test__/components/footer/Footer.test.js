import React from 'react'
import { shallow } from 'enzyme'
import { create } from 'react-test-renderer'
import Footer from '../../../components/footer/Footer'

describe('<Footer/>', () => {
  const footer = shallow(<Footer/>)

  test('Render of the Footer Component', () => {
    expect(footer.length).toEqual(1)
  })
})

describe('<Footer/> Snapshot', () => {
  test('Footer Snapshot test', () => {
    const footer = create(<Footer/>)

    expect(footer.toJSON()).toMatchSnapshot()
  })
}) 