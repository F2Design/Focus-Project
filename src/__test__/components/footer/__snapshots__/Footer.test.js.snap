// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> Snapshot Footer Snapshot test 1`] = `
<footer
  className="footer"
>
  <ul
    className="logo__footer"
  >
    <figure>
      <a
        href="index.html"
      >
        <img
          alt="Logo Focus"
          height="30"
          src="images/logo-focus-footer.svg"
        />
      </a>
    </figure>
  </ul>
  <ul
    className="item__menu-footer"
  >
    <li>
      <a
        href=""
      >
        Soporte
      </a>
    </li>
    <li>
      <a
        href=""
      >
        Nosotros
      </a>
    </li>
  </ul>
  <ul
    className="item__menu-footer"
  >
    <li>
      <a
        href=""
      >
        Noticias
      </a>
    </li>
    <li>
      <a
        href=""
      >
        Privacidad
      </a>
    </li>
  </ul>
  <ul
    className="social__footer"
  >
    <div
      className="social"
    >
      <li
        className="social__icon"
      >
        <a
          href="#"
        >
          <img
            alt="GitHub"
            className=""
            src="images/github.svg"
            width="20"
          />
        </a>
        <p
          className="social__text-footer"
        >
          Github
        </p>
      </li>
      <li
        className="social__icon"
      >
        <a
          href="#"
        >
          <img
            alt="Linkedin"
            className=""
            src="images/linkedin.svg"
            width="20"
          />
        </a>
        <p
          className="social__text-footer"
        >
          Linkedin
        </p>
      </li>
      <li
        className="social__icon"
      >
        <a
          href="#"
        >
          <img
            alt="Facebook"
            className=""
            src="images/facebook.svg"
            width="20"
          />
        </a>
        <p
          className="social__text-footer"
        >
          Facebook
        </p>
      </li>
    </div>
  </ul>
</footer>
`;
